parachain/runtime/runtime-common/src/tests.rs

```diff
@@ -0,0 +1,246 @@
+use crate::XcmExportFeeToSibling;
+use frame_support::{parameter_types, sp_runtime::testing::H256};
+use snowbridge_core::outbound::{Fee, Message, SendError, SendMessage, SendMessageFeeProvider};
+use xcm::prelude::{
+	Here, Kusama, MultiAsset, MultiAssets, MultiLocation, NetworkId, Parachain, XcmContext,
+	XcmError, XcmHash, XcmResult, X1,
+};
+use xcm_builder::HandleFee;
+use xcm_executor::{
+	traits::{FeeReason, TransactAsset},
+	Assets,
+};
+
+parameter_types! {
+	pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 11155111 };
+	pub TokenLocation: MultiLocation = MultiLocation::parent();
+}
+
+struct MockOkOutboundQueue;
+impl SendMessage for MockOkOutboundQueue {
+	type Ticket = ();
+
+	fn validate(_: &Message) -> Result<(Self::Ticket, Fee<Self::Balance>), SendError> {
+		Ok(((), Fee { local: 1, remote: 1 }))
+	}
+
+	fn deliver(_: Self::Ticket) -> Result<H256, SendError> {
+		Ok(H256::zero())
+	}
+}
+
+impl SendMessageFeeProvider for MockOkOutboundQueue {
+	type Balance = u128;
+
+	fn local_fee() -> Self::Balance {
+		1
+	}
+}
+struct MockErrOutboundQueue;
+impl SendMessage for MockErrOutboundQueue {
+	type Ticket = ();
+
+	fn validate(_: &Message) -> Result<(Self::Ticket, Fee<Self::Balance>), SendError> {
+		Err(SendError::MessageTooLarge)
+	}
+
+	fn deliver(_: Self::Ticket) -> Result<H256, SendError> {
+		Err(SendError::MessageTooLarge)
+	}
+}
+
+impl SendMessageFeeProvider for MockErrOutboundQueue {
+	type Balance = u128;
+
+	fn local_fee() -> Self::Balance {
+		1
+	}
+}
+
+pub struct SuccessfulTransactor;
```

Comment:
Prefer this name:

```suggestion
pub struct MockAssetTransactor;
```

Reply:
https://github.com/Snowfork/snowbridge/pull/1109/commits/122572d74f3e49f6195754f0f81191b8ffafecd0